e2e-tests/constants/examples/phone-number-examples.js

```diff
@@ -0,0 +1,21 @@
+export const INVALID_PHONE_NUMBERS = {
+  LONG_LANDLINE: '020727180101',
```

Comment:
Very minor but could we have `LANDLINE` first, like the others

Reply:
done